contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public strategist;
+    mapping(address => bool) public supportedStrategies;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event StrategistUpdated(address strategist);
+    event SupportedStrategyUpdated(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- MODIFIERS
+    ////////////////////////////////////////////////////
+    modifier onlyStrategist() {
+        require(msg.sender == strategist || isGovernor(), "Not strategist");
+        _;
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTRUCTOR
+    ////////////////////////////////////////////////////
+    constructor(address _governor, address _strategist) {
+        _setStrategist(_strategist);
+        _changeGovernor(_governor);
+    }
+
+    ////////////////////////////////////////////////////
+    /// --- MUTATIVE FUNCTIONS
+    ////////////////////////////////////////////////////
+    function harvestAndTransfer(address _strategy) external {
+        _harvestAndTransfer(_strategy);
+    }
+
+    function harvestAndTransfer(address[] calldata _strategies) external {
+        for (uint256 i = 0; i < _strategies.length; i++) {
+            _harvestAndTransfer(_strategies[i]);
+        }
+    }
+
+    function _harvestAndTransfer(address _strategy) internal {
+        // Ensure strategy is supported
```

Comment:
Nit: I think `Assure` or `Assert` will be better here instead of `Ensure`. I think the difference is that ensure is used when the code would need to make sure that something happens / holds true. 

While in this case we rather check for it and revert in case it is not true. A native english speaker will know for sure :) @DanielVF maybe?

Reply:
I think ensure is flexible enough in English that I didn't think twice about at all on this. In our programming code function names we do use "ensure" to mean that it will check something and correct it if it is not currently correct (Ideally always succeeding), vs require style reverting. 

I don't think that that changing the comment matters here though. It's pretty clear what it means. A perfectly correct comment would be "require" but then we are just duplicating the code. :D